docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -164,44 +165,57 @@ If the DUT is a controlee (e.g., light bulb), the TH spins a reference commissio
 
 If the DUT is a commissioner/controller, the Test TH spins an example accessory that is shipped with the SDK and uses that for the DUT to provision, control and run certification tests.
 
-Refer to <<bringing-up-of-matter-node-dut-for-certification-testing, Section 5, Bringing Up of Matter Node (DUT) for Certification Testing>> to bring up the DUT and then proceed with device testing by referring to <<test-configuration, Section 8, Test Configuration>>.
+Refer to <<bringing-up-of-matter-node-dut-for-certification-testing, Section 5, Bringing Up of Matter Node (DUT) for Certification Testing>> to bring up the DUT and then proceed with device testing by referring to <<test-configuration, Section 7, Test Configuration>>.
```

Comment:
Should this be changed?

Reply:
I think it is correct.
<img width="430" alt="image" src="https://github.com/project-chip/certification-tool/assets/116589806/76cdbada-b98c-4f96-816a-833ee7b740ff">